bot/cogs/sudo.py

```diff
@@ -0,0 +1,130 @@
+            await ctx.message.add_reaction("✅")
+            await self.bot.close()
+
+    @sudo.command()
+    async def restart(self, ctx: Context) -> None:
+        """Restart the bot."""
+        if ctx.author.id in config.devs:
+            await ctx.message.add_reaction("✅")
+            await self.bot.close()
+
+            time.sleep(1)
+
+            os.system("pipenv run start")
+
+    @sudo.command()
+    async def load(self, ctx: Context, extension: str = None) -> None:
+        """Load a cog."""
+        if not extension:
+            extension = self.bot.extension_list
+        else:
+            extension = [f"bot.cogs.{extension}"]
+
+        for ext in extension:
+            try:
+                self.bot.load_extension(ext)
+            except DiscordException:
+                await ctx.send(f"```py\n{traceback.format_exc()}\n```")
+            else:
+                await ctx.send("\N{SQUARED OK}")
+
+    @sudo.command()
+    async def reload(self, ctx: Context, extension: str = None) -> None:
+        if not extension:
+            extension = self.bot.extension_list
+        else:
+            extension = [f"bot.cogs.{extension}"]
+
+        for ext in extension:
+            try:
+                self.bot.unload_extension(ext)
+                self.bot.load_extension(ext)
+            except DiscordException:
+                await ctx.send(f"```py\n{traceback.format_exc()}\n```")
+            else:
+                await ctx.send("\N{SQUARED OK}")
+
+    @sudo.command()
+    async def unload(self, ctx: Context, extension: str = None) -> None:
+        if not extension:
+            extension = self.bot.extension_list
+        else:
+            extension = [f"bot.cogs.{extension}"]
+
+        for ext in extension:
+            try:
+                self.bot.unload_extension(ext)
+            except DiscordException:
+                await ctx.send(f"```py\n{traceback.format_exc()}\n```")
+            else:
+                await ctx.send("\N{SQUARED OK}")
+
+    @sudo.command()
+    async def stats(self, ctx: Context) -> None:
+        """Show full bot stats."""
+        implementation = platform.python_implementation()
+
+        general = textwrap.dedent(
+            f"""
+            • Servers: **`{len(self.bot.guilds)}`**
+            • Commands: **`{len(self.bot.commands)}`**
+            • Members: **`{len(set(self.bot.get_all_members()))}`**
+            • Started: **{tm.stringify_reldelta(datetime.utcnow() - self.bot.start_time)}**
+            """
+        )
+        system = textwrap.dedent(
+            f"""
+            • Python: **`{platform.python_version()} with {implementation}`**
+            • discord.py: **`{discord_version}`**
+            """
+        )
+
+        embed = Embed(title="BOT STATISTICS", color=Color.blue())
+        embed.add_field(name="**❯❯ General**", value=general, inline=True)
+        embed.add_field(name="**❯❯ System**", value=system, inline=True)
```

Comment:
It might be a better idea to define the `value` in here directly, rather than storing it in a separate variable and passing it like that.